tools/c7n_azure/tests_azure/tests_resources/test_container_host.py

```diff
@@ -529,6 +530,34 @@ def test_unload_policy_file_that_was_never_loaded(self,
         # Clean up the file
         os.remove(file_path)
 
+    @patch('c7n_azure.container_host.host.Host.update_event_subscription')
+    @patch('c7n_azure.container_host.host.BlockingScheduler.start')
+    @patch('c7n_azure.container_host.host.Host.prepare_queue_storage')
+    @patch('c7n_azure.container_host.host.Storage.get_queue_client_by_storage_account')
+    @patch('c7n_azure.container_host.host.Storage.get_blob_client_by_uri')
+    @patch('os.unlink')
+    def test_unload_policy_file_without_name(self,
+            os_unlink, yaml_safe_load, _1, _2, _3, _4):
+        host = Host(DEFAULT_EVENT_QUEUE_ID, DEFAULT_EVENT_QUEUE_NAME, DEFAULT_POLICY_STORAGE)
+
+        # Create a bad yaml file (no name field)
+        file_path = tempfile.mktemp(suffix=".yaml")
+        with open(file_path, 'w') as f:
+            f.write("""
+                        policies:
+                          - mode:
+                              type: container-periodic
+                              schedule: '* * * * *'
+                            resource: azure.resourcegroup
+                        """)
+
+        host.load_policy(file_path, host.policies)
```

Comment:
Why is there no error when loading an incorrect policy schema? Does the user know a policy was never loaded?

Reply:
We do log errors for the various failure types when we load the policy but we don't fail on them since this is headless and needs to just keep working for all the other policies.  

The tests for these edge cases were all from scenarios where you have a single `yaml` file that contains both valid and non-valid policies.  When we update or remove that file we go unregister jobs for all the policies that were in it, and there were various ways the lookups could fail if a subset of policies never got loaded in the first place.

Reply:
Silly question (it's been awhile) - but is there an easy way to look up what policies are running in a container? Other than looking at the YAML file?

Reply:
Yeah today you'd basically just look at the storage container to see what YAML is in there, or look at the logs or app insights as it traces out the list and their schedules periodically.  Would be nice if it had a little endpoint you could interrogate.